### PR TITLE
 Fixed spacing between form elements

### PR DESCRIPTION
**Pull request by tofran**
- Removed crazy specificity for (almost) all input types 
- Removed double margin-bottom:

As stated in #106 labels can be used as a surrounding element:

```
<label>
  Another field, this time inside label:
  <input name="email" type="text" id="email" size="25" placeholder="must be unique"/>
</label>
```

This created a double margin bottom if used in 'normal' inputs:

chrome_2017-10-25_16-07-20

- Also fixed inconsistent select margin

### CODE REVIEW

**Comment by 0crat** [comment]
@filfreire please, pay attention to this pull request

**Comment by 0crat** [comment]
Job `gh:yegor256/tacit#108` is in scope.

File: scss/_forms.scss [hunk 1]
```diff
@@ -101,26 +104,9 @@ button[type="submit"] {
     background: darken($blue, 15);
     color: darken($white, 25);
   }
-
 }
 
-input[type="text"],
```
Comment (filfreire) [inline scss/_forms.scss]: @tofran what are the possible implications of removing these specific

File: scss/_forms.scss [hunk 2]
```diff
@@ -142,7 +128,5 @@ input[type="radio"] {
 }
 
 select[multiple] {
-  margin-bottom: 0;
```
Comment (filfreire) [inline scss/_forms.scss]: @tofran this causes inconsistency in select margin?
Can you show examples with and without this rule?

**Comment by tofran** [comment]
Ok first of all sorry,
while making this explanation I understand your issue, and what have I done...

Recapping, the problem is this margin:

20171026_selection_01

By adding...
```
> input{
  margin-bottom: 0;
}
```
in `label {`

The problem would persist, due to the [CSS  specificity rules](https://www.w3.org/TR/selectors/#specificity)

20171026_selection_02

So what I naively done was removing this:
```
input[type="password"],
(...)
-input[type="datetime-local"],
```

In the same way that I messed up with the `select[multiple] ` while trying to fix this:

20171026_selection_06

Thank you and good job for not letting that in :)

Sorry for that.

I think its fixed now, have a [look](https://github.com/tofran/tacit/commit/e31d086d133962519a5a4237072bcb256902ddfb)

**Comment by filfreire** [comment]
@rultor merge

**Comment by filfreire** [comment]
@tofran feel free to submit any more issues or PRs you see fit. Thank you for your contributions 👍

**Pull request merged**